index.html

```diff
@@ -9,29 +9,30 @@
     <body>
         <div class="calculator-container">
             <div class="calculator">
-                <input type="text" class="calculator__output" disabled>
+                <input type="text" class="calculator__output" id="output" maxlength="6" disabled>
                 <div class="calculator__action-bar">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__ce" value="CE">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__seven" value="7">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__eight" value="8">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__nine" value="9">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__div" value="&divide">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__four" value="4">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__five" value="5">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__six" value="6">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__mul" value="&times">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__one" value="1">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__two" value="2">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__three" value="3">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__sub" value="-">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__zero" value="0">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__sum" value="+">
-                    <input type="button" class="calculator-action-bar__item action-bar-item__result" value="=">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__ce" id="ce" value="CE">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__seven action-bar__numbers" value="7">
```

Comment:
I would suggest using singular form for such class names (`.action-bar__number`, `.action-bar__operator` etc.)